### ISSUE TITLE
Sprint 4.1: Expand Data Collection, Test Accuracy, Implement API 

### ISSUE THREAD

**Issue by cfloquetcapstone** [cfloquetcapstone/CCC-410-Capstone-Design#11]
For this sprint it's very important for me to expand my current dataset of images. Now that I have nailed down a solid process to take images with the new backlight and process them in a way that spits out a differential cropped image of just that piece, it should be incredibly accurate.

I've already submitted my paperwork and seller documents to Bricklink for approval to get access to their API, so until then I'm focusing on getting as many pictures as I can.

**Checklist (To Do):**
- [x] Gather 15-20 pictures of each main color for each piece 
- [x] Train algorithm using newly gathered data, testing for overfitting
- [x] Create organization.py script to migrate mass image capture to being ready for preprocessing and immediate training. 
- [x] Clean up dataset and remove any bad images. 
- [x] Adjust training_steps, learning_rate to achieve 96-98% accuracy in predictions
- [x] Begin working on API to fetch data about specific bricks or pieces
- [x] Incorporate fetched data into the final UI that is presented to the user.
